src/Totem/Features/API/SampleData.cs

```diff
@@ -157,6 +162,10 @@ public static string GenerateObject(CaseInsensitiveDictionary<SchemaObject> prop
                     {
                         _jsonDictionary[key] = GenerateInteger(value.Format);
                     }
+                    if (value.Type.EqualsCaseInsensitive("number"))
```

Comment:
**SHOULD**: Use `DataType` enumeration.

**Note:**
Using magic values makes the code less maintainable/refactor friendlier